book_source/source/module-1-introduction-to-python/lesson-2-python-basics/conditionals.md

```diff
@@ -27,9 +27,9 @@ This code prints `B` because the first `if` test fails ( `x < 10` is `False` ) a
 
 Two questions you should consider:  
 
--  What values of     `x`     enter the     `else`     statement?  
+-  What values of `x` enter the `else` statement?  
 
--  Why is it not possible to enter the third block (the second     `elif`     block)?  
+-  Why is it not possible to enter the third block (the second `elif` block)?  
```

Comment:
One thing we could do to simplify this page a bit is use an expandable box to show/hide the answer rather than two slides. Maybe we should just edit this in now to streamline this section a bit?

[Here](https://raw.githubusercontent.com/cse163/book/main/book_source/source/module-3-pandas/lesson-7-introduction-to-pandas/imports.md) is an example:

``````
````{admonition} Output
:class: dropdown

```text
Calling b's fun2
Ending b's fun2
Calling a's fun2
Calling a's fun1
Ending a's fun1
Ending a's fun2
```
Notice that `module_a` 's `fun2` behaves exactly the same as it was defined, even though we defined our own `fun1` and `fun2` in `module_b` (the scopes are different).
````
``````